tests/lms/views/decorators/test_h_api.py

```diff
@@ -162,73 +145,23 @@ def test_it_does_nothing_if_the_feature_isnt_enabled(
         wrapped.assert_called_once_with(pyramid_request, mock.sentinel.jwt)
         assert returned == wrapped.return_value
 
-    def test_it_does_nothing_if_the_course_group_already_exists(
-        self, create_course_group, context, models, pyramid_request, wrapped, hapi_svc
-    ):
-        models.CourseGroup.get.return_value = mock.create_autospec(
-            CourseGroup, instance=True
-        )
-
-        returned = create_course_group(pyramid_request, mock.sentinel.jwt, context)
-
-        hapi_svc.post.assert_not_called()
-        assert not pyramid_request.db.add.called
-        wrapped.assert_called_once_with(pyramid_request, mock.sentinel.jwt)
-        assert returned == wrapped.return_value
-
-    def test_it_posts_to_the_group_create_api(
+    def test_it_calls_the_group_update_api(
         self, create_course_group, context, pyramid_request, hapi_svc
     ):
         create_course_group(pyramid_request, mock.sentinel.jwt, context)
 
-        hapi_svc.post.assert_called_once_with(
-            "groups",
-            {"groupid": "test_groupid", "name": "test_group_name"},
-            "acct:test_username@TEST_AUTHORITY",
+        hapi_svc.patch.assert_called_once_with(
+            "groups/test_groupid", {"name": "test_group_name"}
         )
 
-    def test_it_raises_if_post_raises(
+    def test_it_raises_if_patch_raises(
```

Comment:
Re: my comment about the above test name...let's be consistent (`patch` or `update`...my preference is for `patch`).

Reply:
I think `PATCH` is an implementation detail. If the group updating were changed to be implemented in some other way (e.g. to only ever use `PUT` and never `PATCH`) I wouldn't want to have to change the test name. So I've edited these test names to consistently use "update"